lib/rubycritic/core/analysed_module.rb

```diff
@@ -14,6 +14,7 @@ class AnalysedModule
     attribute :complexity
     attribute :duplication,   Integer, :default => 0
     attribute :methods_count
+    attribute :lines
```

Comment:
This line is needed if we want to have the lines= method which is very useful in testing. I can pull it out if you want and then rework the tests, but I think it's worth leaving in for testing purposes.